### PR TITLE
feat: add ability to edit course's value_per_click

### PR DESCRIPTION
**Pull request by srwang**
This PR allows admin users of Publisher to update value per click. The two fields, value_per_click_usa and value_per_click_international track the in-year value per click of each product.  Ultimately they get indexed into Algolia and allow Algolia to leverage this data to provide ranking of items for our users.

Jira ticket: https://2u-internal.atlassian.net/browse/WS-3203
Associated backend pr: https://github.com/openedx/course-discovery/pull/3531
Screenshot:
Screen Shot 2022-07-26 at 3 40 51 PM

### CODE REVIEW

File: src/components/EditCoursePage/EditCourseForm.jsx
Comment (DawoudSheraz) [inline src/components/EditCoursePage/EditCourseForm.jsx:190]: nit: to remove this.

File: src/components/EditCoursePage/index.jsx
Comment (DawoudSheraz) [inline src/components/EditCoursePage/index.jsx:488]: What happens if a non-staff user is editing the page? This can result in setting these values to None.

**Review by DawoudSheraz — COMMENTED** [review]
- Add description in PR, including the linked ticket
- Add screenshots of the changes
- Link to associated backend PR

File: src/components/EditCoursePage/index.jsx
Comment (srwang) [reply src/components/EditCoursePage/index.jsx:488]: I think it's okay because the default value for these fields is 0.  please let me know if I understand your logic or if I'm still missing something!

**Review by Ali-D-Akbar — APPROVED** [review]
Please wait for Dawoud's approval as well.

**Comment by mphilbrick211** [comment]
Hi @srwang! I am just following up on this PR. If you plan to proceed, you will need to re-run your unsuccessful checks. Please let me know if you have any questions!

**Comment by mphilbrick211** [comment]
Hi @srwang! Friendly ping on this. Please let me know if you plan to pursue this PR. Thank you!

**Comment by mphilbrick211** [comment]
Hi @srwang - closing this PR for now. You may reopen it if you choose to pursue in the future. Thanks!

**Pull request closed without merging**
